components/Sidebar.jsx

```diff
@@ -138,6 +138,7 @@ export default class Sidebar extends Component {
           <li>– <a href="#Builder-avg">avg</a></li>
           <li>– <a href="#Builder-increment">increment</a></li>
           <li>– <a href="#Builder-decrement">decrement</a></li>
+          <li>– <a href="#Builder-optimizerHint">optimizerHint</a></li>
```

Comment:
I'm not sure about the order where to put the method

Reply:
I think the place is fine.